pkg/commands/exec.go

```diff
@@ -114,11 +115,11 @@ func (o *VelaExecOptions) Init(ctx context.Context, c *cobra.Command, argsIn []s
 	if err != nil {
 		return err
 	}
+	o.Env = env
```

Comment:
Why do you move line:121 to L:181. Seems `o.Env = nil` all the time.

Reply:
The application's acquisition logic may be wrong, which will affect the env.